### PR TITLE
DAOS-4780 tests: Adding test to run dbench over dfuse

### PR DESCRIPTION
**Pull request by saurabhtandan**
Adding test to run dbench over dfuse.
As part of this pr following changes have been made.
New folder dbench
New file dbench/dbench.*, utils/dbench_utils.py
Added dbench as requirement for daos-tests package.

Quick-Functional: true
Test-tag: dbench

Signed-off-by: Saurabh Tandan <saurabh.tandan@intel.com>

### CODE REVIEW

File: src/tests/ftest/dbench/dbench.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            Purpose of this test is to mount dfuse and run
+            dbench on top of it.
+
+        Use cases:
+            Create Pool.
+            Create container.
+            Mount dfuse.
+            Run dbench on top of mount point.
+
+        :avocado: tags=all,hw,daosio,medium,ib2,full_regression,dbench
```
Comment (brianjmurrell) [inline src/tests/ftest/dbench/dbench.py:33]: This doesn't seem to fit the newly decided/discussed multi-line tag format.

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
daos.spec changes need an updated `Release:` and `%changelog`.

I don't think you can ask for landing with commit pragmas, other than `Features:` but you should indeed use a `Features: dbench` on this PR since it is not run as part of the regular `pr` testing.

File: src/tests/ftest/dbench/dbench.py (hunk 1 above)
Comment (saurabhtandan) [reply src/tests/ftest/dbench/dbench.py]: Done

**Review by shimizukko — DISMISSED** [review]
LGTM

**Review by mjean308 — APPROVED** [review]
LGTM

**Review by mjean308 — DISMISSED** [review]
lgtm

**Review by shimizukko — APPROVED** [review]
LGTM

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
You need to update `Release:` in addition to `%changelog`.  They should match.

You should also merge master as I think there is an update to daos.spec` that you need to base your changes on.

**Comment by saurabhtandan** [comment]
> You need to update `Release:` in addition to `%changelog`. They should match.
> 
> You should also merge master as I think there is an update to daos.spec` that you need to base your changes on.

Done.

**Review by brianjmurrell — APPROVED** [review]

**Review by shimizukko — APPROVED** [review]
LGTM

**Pull request merged**